src/components/app/AppLocalizationProvider.js

```diff
@@ -29,6 +31,13 @@ class AppLocalizationProviderImpl extends React.PureComponent<Props> {
     setupLocalization(locales);
   }
 
+  componentDidUpdate() {
+    const { direction } = this.props;
+    const primaryLocale = navigator.language;
+    ensureExists(document.documentElement).setAttribute('dir', direction);
```

Comment:
I'm not sure why documentElement can be null with flow... but that shouldn't happen for us.